### PR TITLE
Update JuniperSrxResource.java  

### PR DESCRIPTION
**Pull request by wenwenxiong**
fix "action" will not relace in xml
 if type.equals(SecurityPolicyType.SECURITYPOLICY_EGRESS_DEFAULT)

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@wenwenxiong can you add tests and or test data to this? any combination of
- unit tests
- marvin tests
- test procedure

thanks

**Comment by rodrigo93** [comment]
Hi @wenwenxiong, could you make a Jira for this PR?
Did you make any test?

File: plugins/network-elements/juniper-srx/src/com/cloud/network/resource/JuniperSrxResource.java [hunk 1]
```diff
@@ -2841,8 +2841,9 @@ private boolean manageSecurityPolicy(SecurityPolicyType type, SrxCommand command
                             action = "<permit></permit>";
                         }
 
-                        xml = replaceXmlValue(xml, "action", action);
+                        
                     }
+                    xml = replaceXmlValue(xml, "action", action);
```
Comment (rodrigo93) [inline plugins/network-elements/juniper-srx/src/com/cloud/network/resource/JuniperSrxResource.java]: Why did you put it out of "if-then-else"? 
Shouldn't it only be executed only if "!type.equals(SecurityPolicyType.SECURITYPOLICY_EGRESS_DEFAULT)"?

**Comment by rhtyd** [comment]
LGTM, @wenwenxiong can you rebase against latest master and share state of your PR, thanks

tag:easypr

**Comment by bvbharatk** [comment]
### ACS CI BVT Run
 **Sumarry:**
 Build Number 84
 Hypervisor xenserver
 NetworkType Advanced
 Passed=73
 Failed=0
 Skipped=3

_Link to logs Folder (search by build_no):_ https://www.dropbox.com/sh/yj3wnzbceo9uef2/AAB6u-Iap-xztdm6jHX9SjPja?dl=0


**Failed tests:**

**Skipped tests:**
test_vm_nic_adapter_vmxnet3
test_static_role_account_acls
test_deploy_vgpu_enabled_vm

**Passed test suits:**
test_deploy_vm_with_userdata.py
test_affinity_groups_projects.py
test_portable_publicip.py
test_vpc_vpn.py
test_over_provisioning.py
test_global_settings.py
test_scale_vm.py
test_service_offerings.py
test_routers_iptables_default_policy.py
test_routers.py
test_reset_vm_on_reboot.py
test_snapshots.py
test_deploy_vms_with_varied_deploymentplanners.py
test_login.py
test_list_ids_parameter.py
test_public_ip_range.py
test_multipleips_per_nic.py
test_regions.py
test_affinity_groups.py
test_network_acl.py
test_pvlan.py
test_volumes.py
test_nic.py
test_deploy_vm_root_resize.py
test_resource_detail.py
test_secondary_storage.py
test_vm_life_cycle.py
test_disk_offerings.py

**Comment by cloudmonger** [comment]
### ACS CI BVT Run
 **Sumarry:**
 Build Number 821
 Hypervisor xenserver
 NetworkType Advanced
 Passed=108
 Failed=5
 Skipped=12

_Link to logs Folder (search by build_no):_ https://www.dropbox.com/sh/r2si930m8xxzavs/AAAzNrnoF1fC3auFrvsKo_8-a?dl=0

**Failed tests:**
* test_non_contigiousvlan.py

 * test_extendPhysicalNetworkVlan Failed

* test_routers_network_ops.py

 * test_01_isolate_network_FW_PF_default_routes_egress_true Failing since 14 runs

 * test_02_isolate_network_FW_PF_default_routes_egress_false Failing since 14 runs

 * test_01_RVR_Network_FW_PF_SSH_default_routes_egress_true Failing since 14 runs

 * test_02_RVR_Network_FW_PF_SSH_default_routes_egress_false Failing since 14 runs


**Skipped tests:**
test_vm_nic_adapter_vmxnet3
test_01_verify_libvirt
test_02_verify_libvirt_after_restart
test_03_verify_libvirt_attach_disk
test_04_verify_guest_lspci
test_05_change_vm_ostype_restart
test_06_verify_guest_lspci_again
test_static_role_account_acls
test_11_ss_nfs_version_on_ssvm
test_nested_virtualization_vmware
test_3d_gpu_support
test_deploy_vgpu_enabled_vm

**Passed test suits:**
test_deploy_vm_with_userdata.py
test_affinity_groups_projects.py
test_portable_publicip.py
test_vm_snapshots.py
test_over_provisioning.py
test_global_settings.py
test_scale_vm.py
test_service_offerings.py
test_routers_iptables_default_policy.py
test_loadbalance.py
test_routers.py
test_reset_vm_on_reboot.py
test_deploy_vms_with_varied_deploymentplanners.py
test_network.py
test_router_dns.py
test_login.py
test_deploy_vm_iso.py
test_list_ids_parameter.py
test_public_ip_range.py
test_multipleips_per_nic.py
test_metrics_api.py
test_regions.py
test_affinity_groups.py
test_network_acl.py
test_pvlan.py
test_volumes.py
test_nic.py
test_deploy_vm_root_resize.py
test_resource_detail.py
test_secondary_storage.py
test_vm_life_cycle.py
test_disk_offerings.py

**Comment by rhtyd** [comment]
@wenwenxiong any update?

**Comment by DaanHoogland** [comment]
this fix was added to master by means of another commit, closing

**Pull request closed without merging**